src/extension/alterschema/linter/modern_official_dialect_with_empty_fragment.h

```diff
@@ -0,0 +1,36 @@
+class ModernOfficialDialectWithEmptyFragment final
+    : public SchemaTransformRule {
+public:
+  ModernOfficialDialectWithEmptyFragment()
+      : SchemaTransformRule{
+            "modern_official_dialect_with_empty_fragment",
+            "The official dialect URI of Draft 2019-09 and newer versions must "
```

Comment:
```suggestion
            "The official dialect URI of 2019-09 and newer versions must "
```

We don't say draft anymore for the newer ones!